Jenkinsfile.github

```diff
@@ -33,6 +33,8 @@ pipeline {
 
     options {
         buildDiscarder(logRotator(numToKeepStr: "10"))
+        disableConcurrentBuilds()        
```

Comment:
I actually like separate PRs for these sorts of unrelated things in case things go unexpectedly wrong.

Reply:
Ok, I can move it to it's own PR. No worries.